sos/report/plugins/ceph_mgr.py

```diff
@@ -47,12 +47,14 @@ def setup(self):
         microceph_pkg = self.policy.package_manager.pkg_by_name('microceph')
 
         ceph_mgr_cmds = ([
-            "balancer status",
+            "balancer eval",
+            "balancer status detail",
```

Comment:
Curious, as a non ceph person, could it be beneficial of having both `status` and `status detail`?

Reply:
yep. For some reason, I didn't think of it! Included both now.